### DISCUSSION TITLE
Any plans to use an alternative to open.ai GPT ?

### DISCUSSION THREAD

**Discussion by razvanab** [reworkd/AgentGPT#85] [Ideas]
Any plans to use an alternative to open.ai GPT like LLaMA, Alpaca, Vicuna...?

**Comment by colonel11** [comment]
Strongly agree，Claude is also good

**Reply by alexk1919** [reply]
This uses Langchain so you should be able to use other models available from Langchain.

**Reply by colonel11** [reply]
> This uses Langchain so you should be able to use other models available from Langchain.

Got it. Thanks for your explanation.

**Reply by Melvinroy** [reply]
@colonel11 : So Can I use  any  open source/hugging face model instead of open.ai ?
